libraries/AP_OSD/AP_OSD.cpp

```diff
@@ -176,6 +179,7 @@ AP_OSD::AP_OSD()
     osd_type.set_default(HAL_OSD_TYPE_DEFAULT);
 #endif
     previous_pwm_screen = -1;
+    _singleton = this;
```

Comment:
Need a sanity check for only-one-singleton.

Reply:
Ok